staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1365,9 +1367,34 @@ func (e *Store) CompleteWithOptions(options *generic.StoreOptions) error {
 		)
 	}
 
+	if opts.CountMetricPollPeriod > 0 {
+		stopFunc := e.startObservingCount(e.DefaultQualifiedResource.String(), prefix, opts.CountMetricPollPeriod)
```

Comment:
> don't assume prefix is the key... call e.KeyRootFunc() to get the key

this is still outstanding

Reply:
also, no need to pass DefaultQualifiedResource and the output of KeyRootFunc as args, startObservingCount can derive those internally

Reply:
Should I change the decorator call above as well?